### ISSUE TITLE
Ability to generate documentation from multiple locations under pseudo namespaces

### ISSUE THREAD

**Issue by rijnhard** [ApiGen/ApiGen#273]
In some projects, particularly crap ones inherited from monkeys, you have a structure thats not always conducive to document generation. 

We have that situation, we are rewrite the components into a well documented symfony structure. However ever because of the strangeness of the legacy code we cannot generate documentation from it adequately. 

What would be beneficial is if we could specify certain folders to group under a Psuedo namespace, and all the docs from that folder will be grouped under this namespace. 

That way we can still get basic documentation from the legacy code, and not have it clutter the new code. 

Basically the suggestion it to specify multiple locations for document generation, or to specify the entire project and then specify certain folders for exclusion and certain folders for grouping under a pseudo namespace.
